Library/ViewModels/PledgeAmountViewModel.swift

```diff
@@ -158,11 +158,10 @@ public final class PledgeAmountViewModel: PledgeAmountViewModelType,
       .map { ($0.0, $0.1) }
```

Comment:
Should we just map to the max amount because we're not using the minimum amount anymore.